src/OrchardCore.Modules/OrchardCore.Tenants/Workflows/Activities/SetupTenantTask.cs

```diff
@@ -19,14 +19,11 @@ public class SetupTenantTask : TenantTask
     {
         private readonly IClock _clock;
         private readonly IUpdateModelAccessor _updateModelAccessor;
-        private readonly IWorkflowExpressionEvaluator _expressionEvaluator;
-
         public SetupTenantTask(IShellSettingsManager shellSettingsManager, IShellHost shellHost, ISetupService setupService, IClock clock, IWorkflowExpressionEvaluator expressionEvaluator, IWorkflowScriptEvaluator scriptEvaluator, IUpdateModelAccessor updateModelAccessor, IStringLocalizer<SetupTenantTask> localizer)
```

Comment:
Also remove the`expressionEvaluator` from the constructor

Reply:
cant remove , because thats how its passed to the base class.

Reply:
Oops I forgot that, just remove my comment